charts/be-web/templates/_helpers.tpl

```diff
@@ -0,0 +1,134 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "app.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "app.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "app.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "app.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "app.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Hook selector labels
+*/}}
+{{- define "app.hookSelectorLabels" -}}
+app.kubernetes.io/name: {{ include "app.name" .globals }}
+app.kubernetes.io/instance: hook-{{ .hook.name | replace "_" "-" }}
+{{- end }}
+
+{{/*
+Cron selector labels
+*/}}
+{{- define "app.cronSelectorLabels" -}}
+app.kubernetes.io/name: {{ include "app.name" .globals }}
+app.kubernetes.io/instance: cron-{{ .name | replace "_" "-" }}
+{{- end }}
+
+{{/*
+Worker selector labels
+*/}}
+{{- define "app.workerSelectorLabels" -}}
+app.kubernetes.io/name: {{ include "app.name" .globals }}
+app.kubernetes.io/instance: worker-{{ .name | replace "_" "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "app.baseLabels" -}}
+helm.sh/chart: {{ include "app.chart" . }}
+{{ include "app.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
it's impossible to change the version without changing the chart (see comment in `Chart.yaml`)